### PR TITLE
crawl one page from Huxiu

### PR DESCRIPTION
**Pull request by webeautiful**

### CODE REVIEW

File: crawler/cv/spiders/huxiu.py [hunk 1]
```diff
@@ -29,9 +34,9 @@ def parse(self, response):
         item['created_ts'] = None
         item['updated_ts'] = None
         item['time_str'] = None
-        item['author_name'] = None
-        item['author_link'] = None
-        item['author_avatar'] = None
+        item['author_name'] = sel.css('.box-author-info').css('.author-name a::text').extract()[0]
```
Comment (liushuigs) [inline crawler/cv/spiders/huxiu.py]: we can use extract_first() here

File: crawler/cv/spiders/huxiu.py [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
     def parse(self, response):
-        pass
+        print 'start'.center(100, '=')
+        item = self.parse_page(response)
+        print item['author_link']
+        print 'end'.center(100, '=')
+        yield item
+
+    def parse_page(self, response):
+        sel = response.selector
```
Comment (liushuigs) [inline crawler/cv/spiders/huxiu.py:26]: It seems we can directly use response.css or response.xpath

File: crawler/cv/spiders/huxiu.py [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
+        item = ArticleItem()
+        item['url'] = response.url
+        item['title'] = sel.xpath('//title/text()').extract()[0]
+        item['content'] = sel.css('#article_content').extract()[0]
+        item['summary'] = None
+        item['published_ts'] = None
+        item['created_ts'] = None
```
Comment (liushuigs) [inline crawler/cv/spiders/huxiu.py]: We need created_ts and updated_ts are current utc time. Please just keep them like 36kr crawler.

Comment (webeautiful) [inline crawler/cv/spiders/huxiu.py]: I got it

File: crawler/cv/spiders/huxiu.py (hunk 1 above)
Comment (webeautiful) [inline crawler/cv/spiders/huxiu.py]: Don't know before this, I will try it.

File: crawler/cv/spiders/huxiu.py (hunk 2 above)
Comment (webeautiful) [inline crawler/cv/spiders/huxiu.py:26]: But I prefer to this

File: crawler/cv/spiders/huxiu.py [hunk 4]
```diff
@@ -1,13 +1,55 @@
...
+    #    'ITEM_PIPELINES': {
+    #        'cv.pipelines.a36kr.ArticlePipeline': 300
+    #    }
+    #}
 
     def parse(self, response):
-        pass
+        print 'start'.center(100, '=')
+        item = self.parse_page(response)
+        print item['created_ts']
```
Comment (liushuigs) [inline crawler/cv/spiders/huxiu.py]: please remove these prints, we'll replace them with python logging system, and you can debug with scrapy shell

File: crawler/cv/spiders/huxiu.py [hunk 5]
```diff
@@ -1,13 +1,55 @@
...
+        item['summary'] = None
+        item['published_ts'] = None
+        item['created_ts'] = now_date
+        item['updated_ts'] = now_date
+        item['time_str'] = None
+        item['author_name'] = sel.css('.box-author-info').css('.author-name a::text').extract_first()
+        item['author_link'] = sel.css('.box-author-info').css('.author-name a::attr(href)').extract_first()
+        item['author_avatar'] = sel.css('.box-author-info').css('.author-face img::attr(src)').extract_first()
+        item['tags'] = ','.join(sel.css('.tag-box').xpath(".//li[@class='transition']/text()").extract())
+        item['site_unique_id'] = None
```
Comment (liushuigs) [inline crawler/cv/spiders/huxiu.py]: add site_unique_id here

File: crawler/cv/spiders/huxiu.py [hunk 6]
```diff
@@ -1,13 +1,55 @@
...
+        item = ArticleItem()
+
+        now_date = datetime.datetime.utcnow()
+        now_date = now_date.strftime('%Y-%m-%d %H:%M:%S')
+
+        item['url'] = response.url
+        item['title'] = sel.xpath('//title/text()').extract_first()
+        item['content'] = sel.css('#article_content').extract_first()
+        item['summary'] = None
+        item['published_ts'] = None
```
Comment (liushuigs) [inline crawler/cv/spiders/huxiu.py]: we'd better have this field

Comment (liushuigs) [inline crawler/cv/spiders/huxiu.py]: make sure you convert it to utc time

File: crawler/cv/spiders/huxiu.py (hunk 4 above)
Comment (webeautiful) [inline crawler/cv/spiders/huxiu.py]: Resolved!

File: crawler/cv/spiders/huxiu.py (hunk 6 above)
Comment (webeautiful) [inline crawler/cv/spiders/huxiu.py]: ok

**Pull request merged**